CHANGELOG.md

```diff
@@ -5,6 +5,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+* Create `BakeUnnumberedFigures` to cover cases with caption (minor)
```

Comment:
also -- if this is changing how `BakeFigure` works, this will be a breaking change!